package.json

```diff
@@ -133,7 +133,7 @@
     "rimraf": "^3.0.2",
     "source-map-support": "^0.5.19",
     "tslib": "^2.3.0",
-    "typescript": "^4.0.2",
+    "typescript": "^4.4.3",
```

Comment:
Oh. I thought that we were going from 4.3.5 to 4.4.3... based on #738.

Reply:
The `^` in the range means that any minor/patch version is acceptable. e.g. all same major. The package-lock.json has what the current version of the dependencies resolves to. What you are doing is raising up the floor of the acceptable range.